port/win32/omrtime.c

```diff
@@ -178,12 +177,12 @@ omrtime_nano_time(struct OMRPortLibrary *portLibrary)
 		ticks = (int64_t)GetTickCount();
 	}
 
-	if (PPG_time_hiresClockFrequency == OMRTIME_NANOSECONDS_PER_SECOND) {
+	if (PPG_time_hiresClockFrequency == OMRPORT_TIME_DELTA_IN_NANOSECONDS) {
```

Comment:
Apply coding standard since this line has been modified: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#formatting-conditions.
```suggestion
	if (OMRPORT_TIME_DELTA_IN_NANOSECONDS == PPG_time_hiresClockFrequency) {
```